qa-pipelines/qa-pipeline.yml.erb

```diff
@@ -91,6 +91,15 @@ aliases:
     KUBE_ORGANIZATION: ((organization))
     MAGIC_DNS_SERVICE: ((magic-dns-service))
 
+    # Terraform AKS Configuration
+    ARM_CLIENT_ID: ((arm-client-id))
```

Comment:
If these vars are only needed by the terraform task, they shouldn't go in the common task params

Reply:
I think we should keep just one prams for all tasks. It keeps the qa-pipeline.erb clean. I was planning to put every thing in common params after refactor2 merge.

Reply:
This is only needed for one terraform task, so it should go in that task parameter list. It's not a common param, and it puts unnecessary things in the pipeline definition (after templating).

Reply:
Change requested